2012-06-06  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/es_MX (LC_MONETARY): Set mon_thousands_sep to space
	( ).

---
 localedata/locales/es_MX |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/localedata/locales/es_MX
+++ b/localedata/locales/es_MX
@@ -66,7 +66,7 @@
 int_curr_symbol      "MXN "
 currency_symbol      "$"
 mon_decimal_point    "."
-mon_thousands_sep    ","
+mon_thousands_sep    " "
 mon_grouping         3
 positive_sign        ""
 negative_sign        "-"
